content/blog/duckplyr-1-0-0/index.Rmd

```diff
@@ -121,29 +121,26 @@ The result can finally be materialized to memory, or computed temporarily, or co
 
 ```{r}
 # to memory
-out
+collect(out)
 
 # to a file
 csv_file <- withr::local_tempfile()
-file.size(csv_file)
 compute_csv(out, csv_file)
-file.size(csv_file)
+fs::file_size(csv_file)
 ```
 
 When duckplyr itself does not support specific functionality, it falls back to dplyr.
-For instance, row names are not supported yet:
+For instance, pivoting is not supported yet, still it works thanks to the fallback mechanism.
```

Comment:
I find it potentially confusing since pivoting is tidyr not dplyr.

Reply:
And `nrow()` is base, not dplyr 🙃

Reply:
Row names too.

Reply:
How to avoid the confusion then? Could we use this to highlight how seamless all of this is?

Reply:
row names are a characteristic of the data. You cannot use duckplyr with data that have row names or factors.

Now tidyr would be something like the fallbacks needed for select() etc.